client/src/components/ElementEditor/Header.scss

```diff
@@ -40,4 +40,30 @@
   }
 }
 
+// Draft/modified state icons copied from asset-admin - see GalleryItem.scss
+.element-editor-header__version-state {
+  border: 1px solid $state-draft;
+  border-radius: 100%;
+  bottom: 6px;
+  box-shadow: 0 0 1px .5px $white;
+  display: block;
+  height: 8px;
+  left: 22px;
+  position: relative;
+  width: 8px;
+  z-index: 1;
+}
+
+.element-editor__element {
+  &--draft {
+    .element-editor-header__version-state {
```

Comment:
Isn't this not BEM?

Reply:
Yeah not really. We could compromise and have the versioned state on both parts? I do think it’s important to have it on the element too though

Reply:
Yes I think that would be an appropriate compromise. I can feel it's going to be a new pet peeve of mine if we use BEM but make styles with specificity >`0 1 0`